cfpb/consumerfinance.gov
# Migrate legacy ccdb complaint landing page to jinja

Migrate legacy ccdb complaint landing page to jinja. This page used the legacy base_update.html template.

## Removals

- Unused legacy CF and notification CSS.
- Remove unused CSS from landing-page.css


## Changes

- Move complaint-landing template from django to jinja and remove unused markup.
- Import DS CSS by reference.


## How to test this PR

1. Pull branch and run `gulp clean && gulp build` or equivalent.
2. Compare http://localhost:8000/data-research/consumer-complaints/ to https://www.consumerfinance.gov/data-research/consumer-complaints/
3. Page should be almost identical apart from minor differences:
 - Gap before "Things to know before…" well is larger.
 - Notification has moved from above the hero to below the "Things to know before…" well. (Inspect page and look for `m-notification`.)
 - Tilemap key is a gray instead of black text.
 - Tilemap state popup has larger text. (This and the above are a consequence of something coming in from cfpb-chart-builder and not worth diagnosing I think. The larger text seems more accessible.)


## Screenshots

Before:
<img width="1660" alt="Screen Shot 2021-03-25 at 1 17 20 PM" src="https://user-images.githubusercontent.com/704760/112535706-fd4ed900-8d82-11eb-872f-f4f983da1ede.png">

After:
<img width="1349" alt="Screen Shot 2021-03-25 at 1 19 05 PM" src="https://user-images.githubusercontent.com/704760/112535718-00e26000-8d83-11eb-8c82-7c4e3ef9c0f5.png">

Before:
<img width="398" alt="Screen Shot 2021-03-25 at 2 54 01 PM" src="https://user-images.githubusercontent.com/704760/112535758-0a6bc800-8d83-11eb-8ca3-030692173d04.png">

After:
<img width="466" alt="Screen Shot 2021-03-25 at 2 54 10 PM" src="https://user-images.githubusercontent.com/704760/112535776-0f307c00-8d83-11eb-92ad-08b8c3dd13da.png">

Before:
<img width="539" alt="Screen Shot 2021-03-25 at 2 54 34 PM" src="https://user-images.githubusercontent.com/704760/112535829-1b1c3e00-8d83-11eb-8aa9-818ebccb739f.png">

After:
<img width="532" alt="Screen Shot 2021-03-25 at 2 54 30 PM" src="https://user-images.githubusercontent.com/704760/112535805-15bef380-8d83-11eb-83b0-b8dec0a4f571.png">
